sonic_y_cable/y_cable_base.py

```diff
@@ -17,17 +17,18 @@ class YCableBase(object):
     # values will be retreived
 
     TARGET_NIC = 0
-    TARGET_TOR1 = 1
-    TARGET_TOR2 = 2
+    TARGET_TORA = 1
+    TARGET_TORB = 2
+    TARGET_UNKNOWN = -1
```

Comment:
Suggest moving above `TARGET_NIC` to align in numerical order.

Reply:
fixed